locales/en-US/editor.json

```diff
@@ -1,7 +1,7 @@
 {
     "workspace": {
         "label": {
-            "name": "Name"
+            "name": "Name:"
```

Comment:
Have you checked there isn't someone already using this message in its original form?